### PR TITLE
Draft email

### LINKED ISSUE #269: Email notification after in season draft picks
Owners in a league are emailed after each draft pick

### PR DESCRIPTION
**Pull request by axelclark**
Add email notification when owner drafts in season draft pick
* Automatically email league when an owner makes a pick
* Use as part of Ecto Multi
* Still need to figure out email tests
* Closes #269

### CODE REVIEW

File: lib/ex338/in_season_draft_pick/admin.ex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule Ex338.InSeasonDraftPick.Admin do
```
Comment (credoci) [inline lib/ex338/in_season_draft_pick/admin.ex]: Modules should have a @moduledoc tag.

File: test/lib/ex338/owner/store_test.exs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      league_a = insert(:fantasy_league)
+      league_b = insert(:fantasy_league)
+      team_a = insert(:fantasy_team, team_name: "A", fantasy_league: league_a)
+      team_b = insert(:fantasy_team, team_name: "B", fantasy_league: league_b)
+      user_a = insert_user()
+      user_b = insert_user()
+      insert(:owner, fantasy_team: team_a, user: user_a)
+      insert(:owner, fantasy_team: team_b, user: user_b)
+
+
```
Comment (axelclark) [inline test/lib/ex338/owner/store_test.exs]: There should be no more than 1 consecutive blank lines.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 possible new issues (including those that may have been commented here).
* 7 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/273.

**Comment by axelclark** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 7 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/axelclark/ex338/pulls/273.

**Pull request merged**